src/main/java/carpet/patches/FakePlayerManager.java

```diff
@@ -0,0 +1,14 @@
+package carpet.patches;
+
+import net.minecraft.server.network.ServerGamePacketListenerImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FakePlayerManager {
+    public static List<ServerGamePacketListenerImpl> connections = new ArrayList<>();
```

Comment:
Suggestion: use concurrency-proof containers to handle potential concurrent access in certain situations (e.g. using mods)